bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/views/Views.java

```diff
@@ -29,4 +29,6 @@ public static class Publish {}
      */
     public static class Author extends Publish {}
 
+    public static class Submission extends Publish {}
```

Comment:
Please add java doc for this. The reason for the creation of this View.

Reply:
Sure

Reply:
added java doc